player/src/main/java/com/focusit/jsflight/player/configurations/ScriptsConfiguration.java

```diff
@@ -0,0 +1,217 @@
+    }
+
+    public String getIsBrowserHaveErrorScript()
+    {
+        return isBrowserHaveErrorScript;
+    }
+
+    public void setIsBrowserHaveErrorScript(String isBrowserHaveErrorScript)
+    {
+        this.isBrowserHaveErrorScript = isBrowserHaveErrorScript;
+    }
+
+    public String getIsSelectElementScript()
+    {
+        return isSelectElementScript;
+    }
+
+    public void setIsSelectElementScript(String isSelectElementScript)
+    {
+        this.isSelectElementScript = isSelectElementScript;
+    }
+
+    public String getIsUiShownScript()
+    {
+        return isUiShownScript;
+    }
+
+    public void setIsUiShownScript(String isUiShownScript)
+    {
+        this.isUiShownScript = isUiShownScript;
+    }
+
+    public String getStepProcessorScript()
+    {
+        return stepProcessorScript;
+    }
+
+    public void setStepProcessorScript(String stepProcessorScript)
+    {
+        this.stepProcessorScript = stepProcessorScript;
+    }
+
+    public String getScenarioProcessorScript()
+    {
+        return scenarioProcessorScript;
+    }
+
+    public void setScenarioProcessorScript(String scenarioProcessorScript)
+    {
+        this.scenarioProcessorScript = scenarioProcessorScript;
+    }
+
+    public String getScriptEventHandlerScript()
+    {
+        return scriptEventHandlerScript;
+    }
+
+    public void setScriptEventHandlerScript(String scriptEventHandlerScript)
+    {
+        this.scriptEventHandlerScript = scriptEventHandlerScript;
+    }
+
+    public String getShouldSkipKeyboardScript()
+    {
+        return shouldSkipKeyboardScript;
+    }
+
+    public void setShouldSkipKeyboardScript(String shouldSkipKeyboardScript)
+    {
+        this.shouldSkipKeyboardScript = shouldSkipKeyboardScript;
+    }
+
+    public String getSendSignalToProcessScript()
+    {
+        return sendSignalToProcessScript;
+    }
+
+    public void setSendSignalToProcessScript(String sendSignalToProcessScript)
+    {
+        this.sendSignalToProcessScript = sendSignalToProcessScript;
+    }
+
+    public String getGetWebDriverPidScript()
+    {
+        return getWebDriverPidScript;
+    }
+
+    public void setGetWebDriverPidScript(String getWebDriverPidScript)
+    {
+        this.getWebDriverPidScript = getWebDriverPidScript;
+    }
+
+    public String getIsAsyncRequestsCompletedScript()
+    {
+        return isAsyncRequestsCompletedScript;
+    }
+
+    public void setIsAsyncRequestsCompletedScript(String isAsyncRequestsCompletedScript)
+    {
+        this.isAsyncRequestsCompletedScript = isAsyncRequestsCompletedScript;
+    }
+
+    @Override
+    public void loadDefaults()
+    {
+        // Please, sorry me for this hell, but otherwise it would look very ugly
+        // Here we initialize all uninitialized script fields with default script from resources
+        List<String> extensions = Arrays.asList(".groovy", ".js"); // It would be better, if extensions will be sorted by frequency in resources
```

Comment:
Okay then, by how would I know about what script will be used to initialize a specific field.
what if I would change the field's name, what would happen then?